### ISSUE TITLE
The ability to manage your ban list

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#1713]
### Is your feature request related to a problem? Please describe.

Currently right now you cannot manage your band list

### Describe the solution you'd like

I want an extra category in the security tab of my settings menu to where i can remove entries to my ban list

### Describe alternatives you've considered

Writing down the username of every user I band then searching them up in my contacts list to unban them later

### Requesters

@epicEaston197

**Comment by JackTheFoxOtter** [comment]
Maybe instead of putting this into "Security" there could instead make a new category "Moderation", or "Self Moderation", I think that could fit better.

**Comment by JackTheFoxOtter** [comment]
Either that or renaming "Security" to "Safety & Security"

**Comment by epicEaston197** [comment]
We don't have a safety category yet

**Comment by Banane9** [comment]
> Either that or renaming "Security" to "Safety & Security"

Great, then we can still keep the name as "Sicherheit" in German 😂

**Comment by SilvyPaws** [comment]
> Maybe instead of putting this into "Security" there could instead make a new category "Safety", "Moderation", or "Self Moderation", I think that could fit better.

I disagree, I think it should go in Security, keeps it simple and I actually expected it to be there when I was looking for it.

**Comment by Draconas** [comment]
Giving this a poke, someone ran into this issue today and I think should be looked into again, it's been a bit

**Comment by Frooxius** [comment]
Please don't bump issues, it doesn't help their priority (in fact, it makes it a bit worse) - upvote them instead. 

When we decide to prioritize, we sort by number of votes, not by number of comments.
